LegendasTVProvider.cs

```diff
@@ -177,7 +177,13 @@ private IEnumerable<string> FindIds(SubtitleSearchRequest request, CancellationT
                     foreach (var suggestion in suggestions)
                     {
                         var source = suggestion._source;
-                        if (source.id_imdb == imdbId && (request.ContentType == VideoContentType.Movie ? source.tipo == "M" : true))
+                        if (!int.TryParse(source.id_imdb, out var sourceImdb))
+                            break;
```

Comment:
Shouldn't this be `continue` instead of `break`? If you break the loop on **one** error, it'll stop trying to parse the next suggestion.

```suggestion
                        if (!int.TryParse(source.id_imdb, out var sourceImdb))
                            continue;
```